src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from "react";
+import users from "../../api/users";
```

Comment:
Importing `users` directly in this component violates the checklist guideline: "Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components". Consider passing the necessary user data as props from the parent component instead.